### PR TITLE
Hook the onMessage event only when needed

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This PR fixes signing in with some OKTA deployments that make use of the `postMessage` event to handle some internal stuff.

Candidate for a dot release

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-12756

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -234,7 +234,7 @@ unsigned-android: stop pre-build check-style prepare-android-build ## Build an u
 test: | pre-run check-style ## Runs tests
 	@npm test
 
-build-pr: | can-build-pr stop pre-buil check-style ## Build a PR from the mattermost-mobile repo
+build-pr: | can-build-pr stop pre-build check-style ## Build a PR from the mattermost-mobile repo
```
Comment (enahum) [inline Makefile]: this is just to fix the build, will have to submit it in another PR and then rebase this one

**Comment by seyfert** [comment]
I tested this build on an Android device and I can confirm that Okta login works with this build.
